### PR TITLE
React/week1

### PR DESCRIPTION
**Pull request by mahakarzoun**

### CODE REVIEW

File: react/week1/my-app/src/App.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import "./App.css";
+import TodoList from './components/TodoList'
+
+
+const tasks = [
+  {
+    id: 1,
+    task: 'Get out of bed',
+    deadLine: 'Wed Sep 13 2017'
```
Comment (calm-prog) [inline react/week1/my-app/src/App.js]: You can also generate a date like this using the [Date constructor and calling the toDateString method on it](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toDateString).

You don't necessarily have to change it here, I just wanted to let you know about it.

File: react/week1/my-app/src/components/Main.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+function Main(props) {
+    return (
+        <div>
+            <TodoList list={props.todo} />
+        </div>
+
+    )
+}
+
+export default Main
```
Comment (calm-prog) [inline react/week1/my-app/src/components/Main.js]: I don't see this component present in your component tree (you don't import it anywhere). I guess then it is redundant.

**Review by calm-prog — COMMENTED** [review]
Very good! You nicely separated your code into individual components. Everything is readable and easy to follow. 

Just some small comments below.

File: react/week1/my-app/src/App.js (hunk 1 above)
Comment (mahakarzoun) [reply react/week1/my-app/src/App.js]: thanks

**Pull request merged**